tagstudio/src/core/library.py

```diff
@@ -16,10 +16,10 @@
 from enum import Enum
 import ujson
 
-from src.core.json_typing import JsonCollation, JsonEntry, JsonLibary, JsonTag
-from src.core import ts_core
-from src.core.utils.str import strip_punctuation
-from src.core.utils.web import strip_web_protocol
+from tagstudio.src.core.json_typing import JsonCollation, JsonEntry, JsonLibary, JsonTag
+from tagstudio.src.core.constants import TS_FOLDER_NAME, BACKUP_FOLDER_NAME, COLLAGE_FOLDER_NAME, VERSION, TEXT_FIELDS
+from tagstudio.src.core.utils.str import strip_punctuation
+from tagstudio.src.core.utils.web import strip_web_protocol
```

Comment:
`Tagstudio.src...` import format breaks at runtime for vscode and when running from the terminal

```suggestion
from src.core.json_typing import JsonCollation, JsonEntry, JsonLibary, JsonTag
from src.core.constants import TS_FOLDER_NAME, BACKUP_FOLDER_NAME, COLLAGE_FOLDER_NAME, VERSION, TEXT_FIELDS
from src.core.utils.str import strip_punctuation
from src.core.utils.web import strip_web_protocol
```